dev/js/actions/commons/login.js

```diff
@@ -111,56 +111,6 @@ export const googleLogin = (data, dispatch, cb) => {
     });
 };
 
-export const linkedinLogin = (data, dispatch, cb) => {
```

Comment:
bro always comment out the code, so that u don't have to write it again in future, Plz for all code just comment out, don't delete anything.Plz do this in your whole PR